src/query/builder.js

```diff
@@ -10,7 +10,7 @@ import * as helpers from '../helpers';
 import JoinClause from './joinclause';
 import {
   assign, clone, each, isBoolean, isEmpty, isFunction, isNumber, isObject,
-  isString, isUndefined, tail, toArray
+  isString, isUndefined, tail, toArray, concat, isArray
```

Comment:
This PR won't be merged anyway. `increment` / `decrement` was no issue, but seems `insert` will be tricky to make work with this. Will have to rethink this before attempting again.